base_layer/wallet/src/output_manager_service/handle.rs

```diff
@@ -425,6 +451,46 @@ impl OutputManagerHandle {
         }
     }
 
+    pub async fn create_claim_sha_atomic_swap_transaction_with_output(
```

Comment:
What is the purpose of this version of the method? I get the feeling it exists purely for the test? I would prefer you rather perform the test using a base node mock.